doc/dox_comments/header_files/ecc.h

```diff
@@ -636,7 +636,8 @@ int wc_ecc_free(ecc_key* key);
 
     \brief This function frees the fixed-point cache, which can be used
     with ecc to speed up computation times. To use this functionality,
-    FP_ECC (fixed-point ecc), should be defined.
+    FP_ECC (fixed-point ecc), should be defined. Threaded applications should
+    call this function upon exit.
```

Comment:
I think it would be nice to notice this aswell in the manual. As a user of wolfSSL I would look for the "Thread Safety" section if I would implement a multi-threaded server: https://www.wolfssl.com/documentation/manuals/wolfssl/chapter09.html#thread-safety

Reply:
Great idea! I added this blurb to that section:
>Some optimizations allocate memory on a per thread basis. If fixed point ECC cache is enabled (FP_ECC), then threads should release the cached buffers using wc_ecc_free() before the thread exits.

Reply:
I'm not seeing this pushed to manual anywhere. Can you please check if you opened the PR to add this?